CONTRIBUTING.md

```diff
@@ -19,12 +19,31 @@ Follow either of the two links above to access the appropriate CLA and instructi
 
 This repository is used by Chart developers for maintaining the official charts for Kubernetes Helm. If your issue is in the Helm tool itself, please use the issue tracker in the [kubernetes/helm](https://github.com/kubernetes/helm) repository.
 
+### Repository Structure
```

Comment:
There's a section on Repo structure in README.md, do you want to move this there?